python/ray/worker.py

```diff
@@ -69,6 +68,12 @@
 logger = logging.getLogger(__name__)
 
 
+# Visible for testing.
+def _unhandled_error_handler(e: Exception):
+    logger.error("Unhandled error (suppress with "
+                 "RAY_IGNORE_UNHANDLED_ERRORS=1): {}".format(e))
```

Comment:
Instead of `unhandled error` it might be clearer if we explicitly say something like:
`Exception from ObjectRef that was never retrieved with ray.get()`

Reply:
Yeah it's kind of long though... I don't suppose "ungetted error" is a real word?